sql/codegen_xgboost_test.go

```diff
@@ -20,6 +20,35 @@ import (
 	"testing"
 )
 
+const (
+	testXGTrainSelectIris = `
+SELECT *
+FROM iris.train
+TRAIN XGBoostEstimator
+WITH
+	objective = "multi:softmax",
+	num_class = 3,
+	max_depth = 5,
+	eta = 0.3,
+	tree_method = "approx",
+	num_round = 30
+COLUMN sepal_length, sepal_width, petal_length, petal_width
+LABEL class INTO sqlflow_models.my_xgboost_model;
+`
+
+	testXGPredSelectIris = `
+SELECT *
+FROM iris.test
+PREDICT iris.predict
+WITH
+	append_columns = [sepal_length, sepal_width, petal_length, petal_width],
```

Comment:
Does `prob_column `, `detail_column` is required? Can we add these columns by default, so the WITH statment can be shorter...

Reply:
Prob_column, detail_column, leaf_column and append_columns are optional. Only result_column is required for prediction task, which has a default column field: "result".

Reply:
Can keep consistent with tf exmaple, specific result column in `PREDICT iris.predict.result`?